Libraries/Database/SchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using YesSql.Sql;
+
+namespace Lombiq.HelpfulLibraries.Libraries.Database
+{
+    public static class SchemaBuilderExtensions
+    {
+        private const string DocumentId = nameof(DocumentId);
+
+        public static void CreateDocumentIdIndex<T>(this ISchemaBuilder schemaBuilder)
+            where T : class => schemaBuilder.AlterTable(typeof(T).Name, table => table

```

Comment:
Miért van szükség a `where T : class`-ra? Szerintem fölösleges.

A nyíl után mindig legyen sortörés ha a kifejezés többsoros. Általánosságban azt szeretnénk, hogy a behúzás mindig csak a kód tartalmától függjön és ne a benne levő szöveg hosszától. Például ha átneveznénk a `T` paramétert `TIndex`-re akkor emiatt ne változzon 3 másik sor is fölöslegesen.

Reply:
`        public static void CreateDocumentIdIndex<T>(this ISchemaBuilder schemaBuilder) =>
            schemaBuilder.AlterTable(typeof(T).Name, table => table
                                 .CreateIndex(
                                     $"IDX_{typeof(T).Name}_{DocumentId}",
                                     DocumentId));`
Így jó lesz formailag?

Reply:
Nem kell az a nagy behúzás a `.CreateIndex(`-hez se. 

```csharp
public static void CreateDocumentIdIndex<T>(this ISchemaBuilder schemaBuilder) =>
    schemaBuilder.AlterTable(typeof(T).Name, table => table
        .CreateIndex(
            $"IDX_{typeof(T).Name}_{DocumentId}",
            DocumentId));